internal/cmd/marketplace/list.go

```diff
@@ -87,6 +87,7 @@ func listMarketplaceItems(client *client.APIClient, companyID string) error {
 	table.SetCenterSeparator("")
 	table.SetColumnSeparator("")
 	table.SetRowSeparator("")
+	table.SetAutoWrapText(false)
```

Comment:
I don't get it why we want to remove the autowrap feature. How big is as a performance hit?

Reply:
I set AutoWrap to false in order to allow the columns to expand without a new line.  
In this case, we need to avoid a new line in case of a long resource name.

Do you have other ways to achieve the same result?

Reply:
But what happen on a small terminal? the line will be truncated?

Reply:
Not truncated, it will go on a new line in weird way.

I think that using AutoWrap in combination with SetColMinWidth could be a good solution.  
What do you think about that?